FAIL: test_alert_text_length (tests.test_alerts.TestAlerts)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/openpilot/selfdrive/controls/tests/test_alerts.py", line 45, in test_alert_text_length
    self.assertLessEqual(w, MAX_TEXT_WIDTH, msg=msg)
AssertionError: 2200 not less than or equal to 1620 : type: preLaneChangeLeft msg: Checking blind spots preparing to change lanes Left
Try a shorter text

--- a/selfdrive/controls/lib/alerts.py
+++ b/selfdrive/controls/lib/alerts.py
@@ -282,15 +282,15 @@ ALERTS = [
       Priority.LOW, VisualAlert.none, AudibleAlert.none, .1, .1, .1),
   Alert(
       "preLaneChangeLeft",
-      "Checking blind spots preparing to change lanes Left",
-      "Monitor Other Vehicles",
+      "Checking Blind Spots",
+      "Monitor other vehicles left lane change initiated",
       AlertStatus.normal, AlertSize.mid,
       Priority.LOW, VisualAlert.steerRequired, AudibleAlert.none, .0, .1, .1, alert_rate=0.75),
 
   Alert(
       "preLaneChangeRight",
-      "Checking blind spots preparing to change lanes Right",
-      "Monitor Other Vehicles",
+      "Checking Blind Spots",
+      "Monitor other vehicles right lane change initiated",
       AlertStatus.normal, AlertSize.mid,
       Priority.LOW, VisualAlert.steerRequired, AudibleAlert.none, .0, .1, .1, alert_rate=0.75),
 

